src/Crocodealer/Config.hs

```diff
@@ -0,0 +1,25 @@
+module Crocodealer.Config
+  ( LabelRule
+  , Label
+  , Package
+  , Config
+  ) where
+
+data Label = Label
+  { name        :: String
+  , description :: String
+  , color       :: String -- Data.Colour.sRGB24read can convert string specifications like "#00aaff" or "00aaff" 
+  }
+
+data LabelRule
+  = Create
+  | Override
+
+type Package = String
+
+data Config = Config
```

Comment:
I would propose to separate these two concepts in two different modules:

1. `Crocodealer.Core.Label` for all label types and pure functions
2. `Crocodealer.Config` for the configuration data type and it's codec.